cardano-node/cardano-node.cabal

```diff
@@ -1,7 +1,7 @@
 cabal-version: 3.0
 
 name:                   cardano-node
-version:                1.35.4
+version:                1.35.5
```

Comment:
Shouldn't we be bumping all the packages cc: @disassembler

Reply:
Done